### DISCUSSION TITLE
How to change Node version (with NVM) used by runner?

### DISCUSSION THREAD

**Discussion by ghost** [orta/vscode-jest#1041] [Q&A]
Hi, I'm using NVM to handle node versions in my machine. SInce I have some projects running on Node 14 and some on Node 16 I was wondering how to set up my vscode workspace in order to Jest to use the correct node engine for the project?

**Comment by nbolton** [comment]
I'm also wondering about this. I'm trying to use `fnm` to achieve this, but keep getting stuck with the extension.
- https://github.com/jest-community/vscode-jest/issues/1081
- https://github.com/jest-community/vscode-jest/issues/1059
